addon/query-cache.js

```diff
@@ -126,6 +126,9 @@ export default class QueryCache {
     if (options.params) {
       ajaxOptions.data = options.params;
     }
+    if (options.adapterOptions && options.adapterOptions.headers) {
```

Comment:
We need to pass the headers in the other return when the adapter specifies its own `queryURL` (also needs a test for that case)

Reply:
we need a test but we are already passing the headers because we are passing all of options

Reply:
added test when adapter specifies its own queryURL